lib/rbac/src/jwt.rs

```diff
@@ -0,0 +1,13 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, PartialEq, Debug)]
+pub struct Claims {
+    /// Expiration time
+    pub exp: Option<u64>,
+
+    /// Read access
+    pub r: Option<bool>,
+
+    /// Read-write access
+    pub rw: Option<bool>,
```

Comment:
maybe one bool would be enough?
How it can be write access without read?

Reply:
So only read-only / write access claim? Makes sense